build/repo.props

```diff
@@ -1,23 +1,7 @@
-<Project>
-  <ItemGroup>
-    <PackageLineup Include="Internal.AspNetCore.Universe.Lineup" Version="2.1.0-*" />
-    <PackageLineup Include="Internal.AspNetCore.Partners.Lineup" Version="2.1.0-*" />
-  </ItemGroup>
-
+﻿<Project>
   <PropertyGroup>
-    <NuGetPublishFeed>https://dotnet.myget.org/f/aspnetcore-ci-dev</NuGetPublishFeed>
```

Comment:
Are we still publishing the templates to this NuGet feed somehow?

Reply:
It gets pushed by the aspnet/Universe build. https://github.com/aspnet/Universe/blob/dev/build/push.targets